### DISCUSSION TITLE
Fooocus + Colab + Colab Pro+

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2328] [General]
Hi, I use Colab Pro+, and it sometimes (often) is super slow.
T4 GPU, Python 3 (basic settings)
Styles (all at the same time): 
FooocusV2
Fooocus Enhance
Fooocus Sharp 
Fooocus Cinematic
SAI Photographic
SAI Cinematic
SAI Enhance
What to do (explain in fingers please) for basic user. I use it to generate images based on stock photos, nothing special, regular images with realistic views.

**Comment by mashb1t** [comment]
It's a T4 instance, don't expect too much from it.
One tip: reduce the amount of styles to ~2-3 for better prompt alignment and instruction following.
